pyproject.toml

```diff
@@ -15,8 +15,9 @@ classifiers = [
 requires-python = ">=3.6"
 dependencies = [
     "aioconsole>=0.3.1",
-    "aiohttp>=3.7.2",
+    "aiohttp>=3.9.1",
     "websockets>=8.1",
+    "async-timeout>=4.0.3",
```

Comment:
```suggestion
```
I think this is a dependency of aiohttp already, so we do not need to specify it. Did that change?

Reply:
It is no longer a dependency of aiohttp for 3.11 and above, so we have to add it to our own dependencies to ensure we have everything we need for running on those versions.

Reply:
oh, right! thanks!